mobile/lib/common/scaffold_with_nav_bar.dart

```diff
@@ -25,7 +25,7 @@ class ScaffoldWithNavBar extends StatelessWidget {
         appBar: const PreferredSize(
             preferredSize: Size.fromHeight(40), child: SafeArea(child: AppBarWrapper())),
         bottomNavigationBar: BottomNavigationBar(
-          backgroundColor: Colors.white,
+          backgroundColor: const Color(0xFFFAFAFA),
```

Comment:
That's not grey, that's FFFAFAFA!

Reply:
Its copied from the color palette "light", didn't figure it out how to access it from there. 🙈